resources/dicts/events/lifegen_events/murder.json

```diff
@@ -307,25 +308,27 @@
+        "v_c... no matter how many problems you solve, or how many cats you bring together, v_c is always there to mess it all up. It almost feels like they're taunting you at this point, doing this just to anger you. Well, it worked, and now they're dead on the ground in front of you... anything to solve a problem for your clan, even if v_c was the problem.",
+        "No matter how many relationships you sabotage, v_c always, somehow, brings those cats back together again. Thankfully, they won't be able to do that anymore."
     ],
     "murder mediator deputy": [
         "v_c was always a problem-causer, and frankly you have no idea how they were chosen as deputy. They're constantly so confrontational against others, always quick to start fights, and you fear what would happen if they became leader. To ensure your clan never has to deal with that, you invite v_c out on a stroll- straight to the den of a badger. As the angry badger comes out of it's den, you shove v_c down and run as they're left to the badger's claws.",
         "Though mediating is your job, you're tired of seeing the same cat come up again and again. You decide that the best way to stop all these arguments is to... remove the problem. As you lead v_c away, telling them that someone wanted to talk to them, the clueless deputy is entirely unaware of where you're leading them. You feign a confused expression when no other cat shows up to this empty fox den, and then harshly push v_c into the den and roll a rock over it, collapsing the inside. Problem solved."
     ],
     "murder mediator leader": [
-        "With a quiet meow, you dip your head as you enter the leader's den. You were summoned here to deal with an issue that the leader had been having, needing advice with some issues concerning another clan. As you settle down, v_c begins to explain the issue, getting so into it they don't notice you slip deathberries into a piece of prey. They take a small bite, and soon collapse to the ground from the weakness. It doesn't matter how many lives you need to take, you'll take all of them to get them out of the way."
+        "With a quiet meow, you dip your head as you enter the leader's den. You were summoned here to deal with an issue that the leader had been having, needing advice with some issues concerning another clan. As you settle down, v_c begins to explain the issue, getting so into it they don't notice you slip deathberries into a piece of prey. They take a small bite, and soon collapse to the ground from the weakness. It doesn't matter how many lives you need to take, you'll take all of them to get them out of the way.",
+        "While the entire clan is asleep, you sneak into v_c's den and take care of them, losing track of their lives after the fourth."
```

Comment:
"clan" should be "Clan"

"losing track of their lives after the fourth."
to
"though you quickly lose track of their lives before they're fully gone."